policyengine_us/variables/household/expense/person/k12_tuition_and_fees.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class k12_tuition_and_fees(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "K-12 Tuition and fees (from Form 8917)"
+    unit = USD
+    definition_period = YEAR
+
+    formula = sum_of_variables(["tuition_and_fees"])
```

Comment:
I think `tuition_and_fees` refers to college. Neither K-12 nor college tuition and fees are in the CPS, so let's remove the formula.